packages/core/contracts/gnosis-zodiac/implementation/OptimisticGovernor.sol

```diff
@@ -34,59 +42,62 @@ contract OptimisticGovernor is Module, Lockable {
 
     event TransactionExecuted(bytes32 indexed proposalHash, uint256 indexed transactionIndex);
 
-    event ProposalDeleted(bytes32 indexed proposalHash, address indexed sender, bytes32 indexed status);
+    event ProposalExecuted(bytes32 indexed proposalHash, bytes32 indexed assertionId);
+
+    event ProposalDeleted(bytes32 indexed proposalHash, bytes32 indexed assertionId);
 
     event SetBond(IERC20 indexed collateral, uint256 indexed bondAmount);
 
     event SetCollateral(IERC20 indexed collateral);
 
-    event SetRules(string indexed rules);
```

Comment:
why this change.

Reply:
indexing string transforms it to its hash in the logs, so unless the observer knows the actual string this does not add much valuable information to others.

Reply:
very interesting. good thing to know!